test/env/default.sh

```diff
@@ -300,6 +300,33 @@ else
     unset SUITE_SPARSE_ROOT
 fi
 
+# ------
+# Ginkgo
+# ------
+
+if [ "$SUNDIALS_PRECISION" != "extended" ]; then
+    # @develop install is GitHub hash 4c3320c9c4e116a2d5aedf9d042b36d1f327a217
+    if [ "$SUNDIALS_CUDA" == "ON" ]; then
+        if [ "$SUNDIALS_INDEX_SIZE" == "32" ]; then
+            export SUNDIALS_GINKGO=ON
+            export GINKGO_ROOT="$(spack location -i ginkgo@develop +cuda %"$compiler")"
+            export GINKGO_BACKENDS="REF;OMP;CUDA"
```

Comment:
I suppose that once we can test on different hardware, we can expand the set of `GINKGO_BACKENDS`?